gbpservice/neutron/nsf/core/main.py

```diff
@@ -0,0 +1,705 @@
+import os
+import time
+import sys
+import copy
+import threading
+
+import eventlet
+eventlet.monkey_patch()
+
+from multiprocessing.queues import Queue
+from Queue import Empty, Full
+from multiprocessing import Process, Queue, Lock
+import multiprocessing as multiprocessing
+
+from neutron.agent.common import config
+from neutron.common import config as common_config
+from neutron import context
+from neutron.agent import rpc as agent_rpc
+
+from gbpservice.neutron.nsf.core import cfg as core_cfg
```

Comment:
nsf everywhere should be nfp
Instead of core_xxx, use nfp_xxx. Will the be more readable?